css-three-column-layout/styles.css

```diff
@@ -0,0 +1,59 @@
+html {
+  background-color: rgb(0, 174, 255);
+}
+
+h2 {
+  font-family: 'Lato', sans-serif;
+  color: white;
+  font-weight: 400;
+  font-size: 28px;
+}
+
+h3 {
+  font-family: 'Lato', sans-serif;
+  color: white;
+}
+
+p {
+  font-family: 'Lato', sans-serif;
+  color: rgba(255, 255, 255, 0.87);
+}
+
+.container {
+  padding-top: 50px;
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.column-full {
+  width: 100%;
+  text-align: center;
+}
+
+.column-one-third {
+  flex-basis: inherit;
+  justify-content: center;
+  text-align: center;
+}
+
+.row {
+  display: flex;
+  padding-bottom: 20px;
+}
+
+.circle {
+  border: white;
+  background-color: white;
+  border-radius: 50%;
+  height: 100px;
+  width: 100px;
+  margin: auto;
```

Comment:
You can remove this line, the `justify-content` property below is taking care of this for you